src/components/ListHeader/ListHeader.js

```diff
@@ -18,6 +19,7 @@ const ListHeader = () => {
       });
     } else if (navigator.clipboard) {
       navigator.clipboard.writeText(url);
+      NotificationManager.success('Get sharing your list with others!', 'Copied to Clipboard!', 3000);
```

Comment:
"success" can also be changed to error etc for different notification types

Reply:
I think we should add a test for this, we'll need to mock out `'react-notifications'`, allowing us to know when it is called.
```js
jest.mock('react-notifications');
```

We can then simulate clicking on the button and check that the text is copied to clipboard and that the notification is shown
```js
describe('without the web share API available', () => {
  const writeText = jest.fn(); // create a mock function for writeText
  beforeEach(() => {
    // set the navigator items up for our test
    global.navigator.share = null;
    global.navigator.clipboard = { writeText };

    // render and click on the share icon
    const { getByTitle } = render(<ListHeader />);
    const shareIcon = getByTitle('Share this list');
    fireEvent.click(shareIcon);
  });

  it('copies the item to the clipboard', () => {
    expect(writeText).toHaveBeenCalledWith('https://thelist.app');
  });

  it('triggers a notification', () => {
    expect(NotificationManager.success).toHaveBeenCalledWith('Get sharing your list with others!', 'Copied to Clipboard!', 3000);
  });
});
```

Reply:
Great, thanks will get a test added :)

Reply:
Not sure where I'm supposed to place "jest.mock('react-notifications');" and in what context. I understand when you make a mock function and assign it to a constant, but how does the test know to utilise this line of code?

Reply:
There's some great documentation about it here, but the short answer is that it replaces every method in `react-notifications` with a mocked version that:
1. doesn't call the actual underlying implementation
2. allows you to check that the methods were called in the way you expect.

https://jestjs.io/docs/en/mock-functions#mocking-modules